### PR TITLE
chore: add vaccine watcher command

### PR DESCRIPTION
**Pull request by eysi09**
**What this PR does / why we need it**:

This command checks for openings at Berlin vaccination centers, using
the API from [Impstoff.Link](https://impfstoff.link/). Obviously this is only useful for people in
Berlin, and only during the vaccination efforts. A big shout out and
thank you to the people behind Impstoff.Link for making this possible.
And of course to everyone working towards getting Berliners vaccinated.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: core/src/commands/get/get-vaccine.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    const heartEmoji = printEmoji("heart", log) ? ` ${printEmoji("heart", log)}` : ""
+    log.info(dedent`
+      Hello! This commmand will check for openings at Berlin's vaccination centers at a
+      ${Math.floor(intervalMs / 1000)} second interval. If it finds one, you'll receive a notification
+      with links to book an appointment.
+
+      Relevant information will also be printed in the terminal.
+
+      Bookings can be done via Doctolib so we recommend that you set up an account with
+      them if you haven already. You can do that at:
```
Comment (10ko) [inline core/src/commands/get/get-vaccine.ts:105]: haven -> haven't

File: core/src/commands/get/get-vaccine.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+      ${link("https://www.doctolib.de/")}
+
+      A very special thanks to Impstoff.Link for providing the APIs for this and of course to
```
Comment (10ko) [inline core/src/commands/get/get-vaccine.ts:109]: impstoff.link -> impfstoff.link.

Also you could add https:// in front of the link so you can click it from the terminal.

File: core/src/commands/get/get-vaccine.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+export class GetVaccineCommand extends Command {
+  name = "vaccine"
+  help = "Check your garden configuration for errors."
+  emoji: "syringe"
+  noProject = true
+
+  description = dedent`
+    Throws an error and exits with code 1 if something's not right in your garden.yml files.
```
Comment (10ko) [inline core/src/commands/get/get-vaccine.ts:86]: I think this is left from the copy pasta. :)

**Review by 10ko — COMMENTED** [review]
Looks great, I just had few minor comments. Also it looks like you have to generate the docs, because the linter is failing. ;) 

 💉 💉 💉

**Comment by eysi09** [comment]
I've updated the PR

**Review by 10ko — APPROVED** [review]
❤️ 💉 ❤️

**Pull request merged**